README.md

```diff
@@ -26,10 +26,9 @@
 az login
 
 # verify the correct subscription
-az account show
-
-# install kubectl and kubelogin
-sudo az aks install-cli
+# you must be the owner of the subscription
+# tenant ID should be 72f988bf-86f1-41af-91ab-2d7cd011db47 
+az account show -o table
```

Comment:
Would it be helpful if we also add command `az account set --subscription ""` ?
With `az login` it defaults to some random subscription

Reply:
added